.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+
+**This is a(n):**
+<!-- choose one by changing [ ] to [x] -->
+- [ ] New feature proposal
+- [ ] Reporting a keyword
```

Comment:
I'm not certain what this one would be, is it a bug? Or a typo?

Reply:
It's basically for bugs